tools/python/src/face_recognition.cpp

```diff
@@ -300,6 +360,11 @@ void bind_face_recognition(py::module &m)
             "If num_jitters>1 then each face will be randomly jittered slightly num_jitters times, each run through the 128D projection, and the average used as the face descriptor. "
             "Optionally allows to override default padding of 0.25 around the face."
             )
+        .def("compute_face_descriptor", &face_recognition_model_v1::compute_face_descriptor_from_aligned_image,
+            py::arg("img"), py::arg("num_jitters")=0,
+            "Takes an aligned face image of size 150x150 and converts it into a 128D face descriptor. "
```

Comment:
Note that the alignment must be done in the way `dlib.get_face_chip` does it.  In particular, it must be 150x150 as well as centered and scaled the same way.